<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[====db connection successful====]]>
    </line>
    <line>
      <![CDATA[====chrome browser====]]>
    </line>
    <line>
      <![CDATA[====browser launch successful===]]>
    </line>
    <line>
      <![CDATA[====login sucessfull====]]>
    </line>
    <line>
      <![CDATA[createContactWithOrganisationAndLeadSource---testscript execution started]]>
    </line>
    <line>
      <![CDATA[createContactWithOrganisationAndLeadSource---testscript execution failed]]>
    </line>
    <line>
      <![CDATA[====logout sucessfull====]]>
    </line>
    <line>
      <![CDATA[====browser close successfull====]]>
    </line>
    <line>
      <![CDATA[====Database close successfull====]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="17507" started-at="2022-03-17T05:33:02Z" finished-at="2022-03-17T05:33:19Z">
    <groups>
      <group name="smokeSuite">
        <method signature="CreateContactWithOrganisationAndLeadSourceTest.createContactWithOrganisationAndLeadSource()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="createContactWithOrganisationAndLeadSource" class="com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest"/>
      </group> <!-- smokeSuite -->
    </groups>
    <test name="Default test" duration-ms="17507" started-at="2022-03-17T05:33:02Z" finished-at="2022-03-17T05:33:19Z">
      <class name="com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest">
        <test-method status="PASS" signature="connectDataBase()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="connectDataBase" is-config="true" duration-ms="22" started-at="2022-03-17T05:33:02Z" finished-at="2022-03-17T05:33:02Z">
          <reporter-output>
            <line>
              <![CDATA[====db connection successful====]]>
            </line>
          </reporter-output>
        </test-method> <!-- connectDataBase -->
        <test-method status="PASS" signature="launchTheBrowser()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="launchTheBrowser" is-config="true" duration-ms="3384" started-at="2022-03-17T05:33:02Z" finished-at="2022-03-17T05:33:06Z">
          <reporter-output>
            <line>
              <![CDATA[====chrome browser====]]>
            </line>
            <line>
              <![CDATA[====browser launch successful===]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchTheBrowser -->
        <test-method status="PASS" signature="login()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="login" is-config="true" duration-ms="3119" started-at="2022-03-17T05:33:06Z" finished-at="2022-03-17T05:33:09Z">
          <reporter-output>
            <line>
              <![CDATA[====login sucessfull====]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="createContactWithOrganisationAndLeadSource()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="createContactWithOrganisationAndLeadSource" duration-ms="8430" started-at="2022-03-17T05:33:09Z" finished-at="2022-03-17T05:33:17Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest.createContactWithOrganisationAndLeadSource(CreateContactWithOrganisationAndLeadSourceTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[createContactWithOrganisationAndLeadSource---testscript execution started]]>
            </line>
            <line>
              <![CDATA[createContactWithOrganisationAndLeadSource---testscript execution failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- createContactWithOrganisationAndLeadSource -->
        <test-method status="PASS" signature="logout()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="logout" is-config="true" duration-ms="1373" started-at="2022-03-17T05:33:17Z" finished-at="2022-03-17T05:33:19Z">
          <reporter-output>
            <line>
              <![CDATA[====logout sucessfull====]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="closeBrowser" is-config="true" duration-ms="694" started-at="2022-03-17T05:33:19Z" finished-at="2022-03-17T05:33:19Z">
          <reporter-output>
            <line>
              <![CDATA[====browser close successfull====]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeDb()[pri:0, instance:com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest@5700d6b1]" name="closeDb" is-config="true" duration-ms="1" started-at="2022-03-17T05:33:20Z" finished-at="2022-03-17T05:33:20Z">
          <reporter-output>
            <line>
              <![CDATA[====Database close successfull====]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeDb -->
      </class> <!-- com.crm.ContactsTest.CreateContactWithOrganisationAndLeadSourceTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
